Sofa/framework/Core/src/sofa/core/objectmodel/Tag.cpp

```diff
@@ -104,9 +61,4 @@ std::istream& operator>>(std::istream& i, Tag& t)
     return i;
 }
 
-
-} // namespace objectmodel
-
-} // namespace core
-
-} // namespace sofa
+}
```

Comment:
Usually in SOFA the namespace is repeated in a comment after the closing curly bracket.

Reply:
Open question: repeating the namespace's name with comment in the ending curlybrace was a trick to improve readability dating back from when it was not possible to have the single line syntax (namespace a::b::c). I'm not sure it is so much needed nowaydays.

Reply:
I totally agree with you and I want to campaign for removing this rule. Meanwhile, you still have discrepancies between the declaration of the namespace and the ending curly bracket comment (in TagSet for example). Either remove the comments, or fix the comments.

Reply:
Sure.